src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,23 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+
+        Machine bulldozer = new Bulldozer();
+        bulldozer.doWork();
+
+        Machine track = new Track();
+        track.doWork();
+
+        Machine excavator = new Excavator();
+        excavator.doWork();
+
+        bulldozer.stopWork();
+
+        track.stopWork();
+
+        excavator.stopWork();
+
+    }
 
```

Comment:
same here :)
too many empty lines make the code look larger and and less readable

Reply:
done